src/styles/stars.css

```
@@ -0,0 +1,52 @@
+.stars {
+  --star-size: 16px;
+
+  display: flex;
+}
+
+.stars__star {
+  flex-shrink: 0;
+  height: var(--star-size);
+  width: var(--star-size);
+  background-image: url(../images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: 13px 12px;
+}
+
+.stars .stars__star {
+  margin-left: 4px;
+}
+
+.stars .stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--1 .stars__star:nth-child(1) {
```

```suggestion
.stars--1 .stars__star:nth-child(1),
... {
```
Consider using commas to group selectors and avoid repeating code